src/Fixer/Basic/BracesFixer.php

```diff
@@ -26,9 +28,45 @@
  *
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
  */
-final class BracesFixer extends AbstractFixer implements WhitespacesAwareFixerInterface
+final class BracesFixer extends AbstractFixer implements WhitespacesAwareFixerInterface, ConfigurableFixerInterface
 {
     /**
+     * @var array
+     */
+    private static $defaultConfiguration = array(
+        'classy_constructs' => 'next',
+    );
+
+    private $config = array();
+
+    /**
+     * Use 'classy_constructs' => 'same'|'next'.
+     *
+     * @param array|null $configuration
+     *
+     * @throws InvalidFixerConfigurationException
+     */
+    public function configure(array $configuration = null)
+    {
+        if (null === $configuration) {
+            $this->config = self::$defaultConfiguration;
+
+            return;
+        }
+
+        if (array_key_exists('classy_constructs', $configuration)) {
+            if (!in_array($configuration['classy_constructs'], array('same', 'next'), true)) {
```

Comment:
At least [ConcatSpace](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/src/Fixer/Operator/ConcatSpaceFixer.php#L33) and [ArraySyntax](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/src/Fixer/ArrayNotation/ArraySyntaxFixer.php#L38) do not. But I agree it is better and will add it.